brave/brave-core
# Removes anonymous address from external wallet flow

Resolves https://github.com/brave/brave-browser/issues/11150

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim ad grant
- connect KYC uphold wallet
- make sure that claim works

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
